internal/rbd/controllerserver.go

```diff
@@ -800,7 +800,7 @@ func (cs *ControllerServer) CreateSnapshot(ctx context.Context, req *csi.CreateS
 		vol := generateVolFromSnap(rbdSnap)
 		err = vol.Connect(cr)
 		if err != nil {
-			uErr := undoSnapshotCloning(ctx, vol, rbdSnap, vol, cr)
+			uErr := undoSnapshotCloning(ctx, rbdVol, rbdSnap, vol, cr)
```

Comment:
The commit " rbd: pass parent volume to undoSnapshotCloning function" only changes the variable names on a few locations, it should probably be squashed with an other one?

Reply:
This PR does not make any changes to `undoSnapshotCloning()` implementation, just passes parent volume to clean up the snapshots on parent volume. This commit covers all the argument changes for a single function call at all places, thus kept in a single commit. Should I leave it as a commit or squash it with another one?